comfyui_custom_nodes/webui_checkpoint_loader.py

```diff
@@ -37,7 +37,9 @@ def INPUT_TYPES(s):
     CATEGORY = "loaders"
 
     def load_checkpoint(self, void):
-        with CheckpointLoaderPatched(shared.sd_model_state_dict) as patched_loader:
+        shared.send_state_dict_event.set()

```

Comment:
Would it be a good idea to have the shared dict not directly on the shared module? I think it's a bit too general. 
Something like this?
```python
shared.comfyui_extension.send_state_dict_event.set()
```